src/test-helpers/typeFactory.ts

```diff
@@ -37,13 +58,29 @@ export class TypeFactory {
 		this.#registry = this.#api.registry;
 	}
 
-	storageKey = (index: number): StorageKey => {
-		const key = new StorageKey(
-			this.#registry,
-			this.#api.query.crowdloan.funds.key(this.paraIndex(index))
-		);
-
-		return key.setMeta(this.#api.query.crowdloan.funds.creator.meta);
+	/**
+	 * @param index
+	 * The id to assign the key to.
```

Comment:
These can go on the same line as the param if you want - vscode at least still understands.

Reply:
Fixed, yea not sure what I was doing putting it below. We always write it to the side!